### ISSUE TITLE
Using Browsix without Node?

### ISSUE THREAD

**Issue by HopperMCS** [plasma-umass/browsix#46]
Is it possible not to rely on Node after initial setup? If so, how? Thanks, this project looks awesome. And, it's also possible that once I accomplish what I want to do with it that I may contribute back.

**Comment by bpowers** [comment]
Sorry for the delay @MGageMorgan. I think any dependence on node is accidental -- I've just removed a few dependencies from `package.json` (node-pipe2 and node-priority) that were initially used to run Browsix tests under node, but are no longer used.  You should be able to use browsix in a web application served off a CDN -- no server necessary.

Let me know if there are more or other specific dependencies on node that you see, and I'll either try to remove them or at least document why they are there.

**Comment by MGageMorgan** [comment]
Thanks, @bpowers - how do I leverage Browsix without some backend server running directly? Is there an API/existing CDN? I have some big things I wanna do with this project, but they're under wraps due to company internal stuff. I'll come back here if that ever finishes.

**Comment by MGageMorgan** [comment]
@bpowers, sorry just re-read what I wrote previously. I worded that soooo wrong. The question should have been "How does one go about setting this up without node at all whatsoever," like, could I just downlaod code and integrate with an existing project out-of-the-box? If we could get Browsix to a near-complete state (e.g. modded package managers for it, lol I know it sounds like a moonshot, but more or less feels like it should be possible) that would be incredibly amazing.
